frontend/app/assets/javascripts/spree/frontend/checkout/address.js

```diff
@@ -59,7 +59,7 @@ Spree.ready(function ($) {
         if (states.length > 0) {
           selected = parseInt(stateSelect.val())
           stateSelect.html('')
-          statesWithBlank = [{name: '', id: ''}].concat(states)
+          statesWithBlank = [{name: 'STATE', id: ''}].concat(states)
```

Comment:
```suggestion
          statesWithBlank = [{name: Spree.translations.country_state. toUpperCase(), id: ''}].concat(states)
```

translations must be added to https://github.com/spree/spree/blob/b862066507361dce9273440809738d14dcd7e3aa/frontend/app/views/spree/shared/_translations.html.erb#L1